src/flap.sol

```diff
@@ -98,13 +106,16 @@ contract Flapper is LibNote {
         if (what == "beg") beg = data;
         else if (what == "ttl") ttl = uint48(data);
         else if (what == "tau") tau = uint48(data);
+        else if (what == "limit") limit = data;
```

Comment:
what about `lid` for proper daiwanese

Reply:
Sure, I don't have strong preference for names.